packages/table/components/Table.tsx

```diff
@@ -39,6 +40,10 @@ export interface TableProps {
    * Selected row data
    */
   selectedRows?: object[];
+  /**
+   * Set the row height
+   */
+  rowHeight?: number;
```

Comment:
Random thought, but we could make bad practices more screaming 🤷‍♂️ 

```suggestion
  dangerouslyOverwriteRowHeight?: number;
```

Reply:
Is it really that dangerous? :D

Reply:
I wouldn't say it's "dangerous" as in it will cause a bug or some unexpected behavior. I just worry about the design tradeoffs of using taller row heights to cram too much information into a single cell